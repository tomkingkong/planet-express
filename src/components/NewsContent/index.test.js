import React from 'react';
import { shallow } from 'enzyme';

import { NewsContent } from '.';

describe('NewsContent component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<NewsContent />);
    expect(wrapper).toMatchSnapshot();
  });
});