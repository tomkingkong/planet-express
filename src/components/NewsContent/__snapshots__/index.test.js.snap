// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsContent component should match snapshot 1`] = `
<article
  className="news__article"
>
  <div
    className="news__img"
    style={
      Object {
        "backgroundImage": "url(undefined)",
      }
    }
  />
  <div
    className="news__info"
  >
    <h4
      className="news__title"
    />
    <p
      className="news__desc"
    />
    <p
      className="news__auth"
    >
      Author: 
    </p>
    <a
      className="news__link"
      target="_blank"
    >
      Link to Article
    </a>
    <p
      className="news__publish"
    />
  </div>
</article>
`;
